WHATS-NEXT.md

```diff
@@ -0,0 +1,5 @@
+# Deploy the blog
+A blog on your local machine is nice to have, but a blog on the internet is even better. [Return to the tutorial doc, select a hosting service (Netlify or Heroku) and follow the CLI to deploy](https://www.contentful.com/developers/docs/tutorials/general/get-started/).
```

Comment:
looks good. later, let's use the markdown anchor 👌 after we've updated the doc.

Reply:
yeah exactly :)

Reply:
I also want to see how the link will be formatted